src/jsifier.mjs

```
@@ -0,0 +1,912 @@
+        }
+        return;
+      }
+
+      // This gets set to true in the case of dynamic linking for symbols that
+      // are undefined in the main module.  In this case we create a stub that
+      // will resolve the correct symbol at runtime, or assert if its missing.
+      let isStub = false;
+
+      const mangled = mangleCSymbolName(symbol);
+
+      if (!LibraryManager.library.hasOwnProperty(symbol)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)